lab4/classes/Video.php

```diff
@@ -0,0 +1,31 @@
+
+<?php
+    include_once(__DIR__ . "/Db.php");
+    class Video {
+
+        public static function getAll() {
+            // this function should get all the video from the database
+            // and return them as an array
+            // tip: use fetchAll() and return that result
+            $conn = \Db::getInstance();
+            $result = $conn->query("select * from videos;");
+            return $result->fetchAll();
```

Comment:
Geef best PDO_ASSOC mee als dat je default is of stel dit als default in op je databank connectie, dan moet je dat nadien niet telkens manueel doen